src/shell/src/command/save.rs

```diff
@@ -40,20 +40,22 @@ impl Command<SaveCommand> {
             Command(
                 InitializedSaveCommand {
                     path:
-                    Self::extract_path_from_args(cwd, args, "path").unwrap_or_else(|_| cwd.to_path_buf().join(".fc.json"))
+                    Self::extract_path_from_args(cwd, args, "path").unwrap_or_else(|_| cwd.to_path_buf().join(".fc.json")),
+                    overwrite: args.is_present("o")
                 }
             )
         )
     }
 }
 
 pub struct InitializedSaveCommand {
-    pub path: PathBuf
+    pub path: PathBuf,
+    pub overwrite: bool
 }
 
 impl Command<InitializedSaveCommand> {
     pub fn execute(&self, fs: &mut Container) -> Result<(), CommandError> { //TODO consume command with "execute(self,"
-        if self.0.path.exists() {
+        if ! self.0.overwrite && self.0.path.exists() {
```

Comment:
I think you must write explicitly the boolean comparison, it's not a big deal and leave the code speaking.

Reply:
there is no `and` / `or` in rust. ( but there is a lot of methods `.and()` / `or.()` )
Personnally i'm agree with you in favor of explicit operators but in another hand i'm happy the language specify only one single way to do that, making anyone's code uniform